### ISSUE TITLE
[Transmission] "walk"-ing deletes torrents

### ISSUE THREAD

**Issue by SakhJack** [idlesign/torrt#56]
Upon using `torrt walk` torrents are deleted instead of updating (files are intact)
<details>
  <summary>Debug `add_torrent`</summary>

  ```console
C:\Windows\system32>torrt add_torrent https://rutracker.org/forum/viewtopic.php?t=5331180 --verbose
DEBUG: Bootstrapping torrt environment ...
DEBUG: Importing rpc ...
DEBUG: Registering `deluge` from <class 'torrt.rpc.deluge.DelugeRPC'> ...
DEBUG: Registering `qbittorrent` from <class 'torrt.rpc.qbittorrent.QBittorrentRPC'> ...
DEBUG: Registering `transmission` from <class 'torrt.rpc.transmission.TransmissionRPC'> ...
DEBUG: Registering `utorrent` from <class 'torrt.rpc.utorrent.UTorrentRPC'> ...
DEBUG: Importing trackers ...
DEBUG: Registering `tr.anidub.com` from <class 'torrt.trackers.anidub.AniDUBTracker'> ...
DEBUG: Registering `anilibria.tv` from <class 'torrt.trackers.anilibria.AnilibriaTracker'> ...
DEBUG: Registering `casstudio.tv` from <class 'torrt.trackers.casstudio.CasstudioTracker'> ...
DEBUG: Registering `kinozal.tv` from <class 'torrt.trackers.kinozal.KinozalTracker'> ...
DEBUG: Registering `nnm-club.me` from <class 'torrt.trackers.nnmclub.NNMClubTracker'> ...
DEBUG: Registering `rutor.org` from <class 'torrt.trackers.rutor.RutorTracker'> ...
DEBUG: Registering `rutracker.org` from <class 'torrt.trackers.rutracker.RuTrackerTracker'> ...
DEBUG: Importing notifiers ...
DEBUG: Registering `email` from <class 'torrt.notifiers.mail.EmailNotifier'> ...
DEBUG: Registering `telegram` from <class 'torrt.notifiers.telegram.TelegramNotifier'> ...
DEBUG: Importing bots ...
DEBUG: Registering `telegram` from <class 'torrt.bots.telegram_bot.TelegramBot'> ...
DEBUG: Initializing objects registries from configuration file ...
DEBUG: Loading configuration file C:\Users\Toyohara\.torrt\config.json ...
DEBUG: Spawning `TransmissionRPC` object with the given settings ...
DEBUG: Registering `transmission` from transmission ...
DEBUG: Spawning `AnilibriaTracker` object with the given settings ...
DEBUG: Registering `anilibria.tv` from anilibria.tv ...
DEBUG: Spawning `RutorTracker` object with the given settings ...
DEBUG: Registering `rutor.org` from rutor.org ...
DEBUG: Spawning `RuTrackerTracker` object with the given settings ...
DEBUG: Registering `rutracker.org` from rutracker.org ...
DEBUG: Adding torrent from `https://rutracker.org/forum/viewtopic.php?t=5331180` ...
DEBUG: Downloading torrent file from `https://rutracker.org/forum/viewtopic.php?t=5331180` ...
DEBUG: RuTrackerTracker: Picking a mirror ...
DEBUG: RuTrackerTracker: Probing mirror: `https://rutracker.org` ...
DEBUG: Starting new HTTPS connection (1): rutracker.org:443
DEBUG: https://rutracker.org:443 "GET / HTTP/1.1" 301 162
DEBUG: https://rutracker.org:443 "GET /forum/index.php HTTP/1.1" 200 None
DEBUG: RuTrackerTracker: Fetching https://rutracker.org/forum/viewtopic.php?t=5331180 ...
DEBUG: Starting new HTTPS connection (1): rutracker.org:443
DEBUG: https://rutracker.org:443 "GET /forum/viewtopic.php?t=5331180 HTTP/1.1" 200 None
DEBUG: RuTrackerTracker: Torrent download link found: https://rutracker.org/forum/dl.php?t=5331180
DEBUG: RuTrackerTracker: Downloading torrent file from https://rutracker.org/forum/dl.php?t=5331180 ...
DEBUG: RuTrackerTracker: Fetching https://rutracker.org/forum/dl.php?t=5331180 ...
DEBUG: Starting new HTTPS connection (1): rutracker.org:443
DEBUG: https://rutracker.org:443 "GET /forum/dl.php?t=5331180 HTTP/1.1" 200 32219
DEBUG: Torrent was downloaded from `https://rutracker.org/forum/viewtopic.php?t=5331180`
DEBUG: TransmissionRPC: RPC method `torrent-add` ...
DEBUG: Starting new HTTP connection (1): 127.0.0.1:9091
DEBUG: http://127.0.0.1:9091 "POST /transmission/rpc HTTP/1.1" 409 581
DEBUG: Starting new HTTP connection (1): 127.0.0.1:9091
DEBUG: http://127.0.0.1:9091 "POST /transmission/rpc HTTP/1.1" 200 145
DEBUG: Registering `298f99aad2c59e7020b9779ea6808d390afa2f95` torrent ...
DEBUG: Loading configuration file C:\Users\Toyohara\.torrt\config.json ...
DEBUG: Saving configuration file C:\Users\Toyohara\.torrt\config.json ...
INFO: Torrent from `https://rutracker.org/forum/viewtopic.php?t=5331180` is added within `transmission`

  ```
</details>
<details>
  <summary>config.json (credentials hidden)</summary>

  ```javascript
{
    "time_last_check": 1598466246,
    "walk_interval_hours": 24,
    "trackers": {
        "rutracker.org": {
            "username": "username",
            "query_string": null,
            "cookies": {
                "bb_session": "session",
                "bb_ssl": "1"
            },
            "password": "password"
        }
    },
    "rpc": {
        "transmission": {
            "url": "http://127.0.0.1:9091/transmission/rpc",
            "enabled": true,
            "host": "127.0.0.1",
            "user": null,
            "password": null,
            "port": "9091"
        }
    },
    "torrents": {
        "c0424991902bdba637dc1647d4b0a2869dbd1b21": {
            "hash": "c0424991902bdba637dc1647d4b0a2869dbd1b21",
            "name": "No Mans Sky by xatab",
            "url": "https://rutracker.org/forum/viewtopic.php?t=5932207",
            "url_file": "https://rutracker.org/forum/dl.php?t=5768151",
            "page": {
                "title": "No Man's Sky [P] [RUS + ENG + 11 / RUS + ENG + 10] (2016, VR Supported) (2.62 + 1 DLC) :: RuTracker.org",
                "cover": "",
                "date_updated": ""
            }
        },
        "1c035226e707d6712674a9048d09c888bfbe81ee": {
            "hash": "1c035226e707d6712674a9048d09c888bfbe81ee",
            "name": "Frostpunk.Game.of.The.Year.Edition.GOG-InsaneRamZes",
            "url": "https://rutracker.org/forum/viewtopic.php?t=5932207",
            "url_file": "https://rutracker.org/forum/dl.php?t=5932207",
            "page": {
                "title": "Frostpunk: Game of the Year Edition [L] [RUS + ENG + 10 / ENG] (2018) (1.6 + 3 DLC) [GOG] :: RuTracker.org",
                "cover": "",
                "date_updated": ""
            }
        },
        "298f99aad2c59e7020b9779ea6808d390afa2f95": {
            "hash": "298f99aad2c59e7020b9779ea6808d390afa2f95",
            "name": "setup_rimworld_1.2.2723",
            "url": "https://rutracker.org/forum/viewtopic.php?t=5331180",
            "url_file": "https://rutracker.org/forum/dl.php?t=5331180",
            "page": {
                "title": "RimWorld [L] [RUS + ENG + 26] (2018) (1.2.2723 + 1 DLC) [GOG] :: RuTracker.org",
                "cover": "",
                "date_updated": ""
            }
        }
    },
    "notifiers": {},
    "bots": {}
}
  ```
</details>
<details>
  <summary>Debug `walk`</summary>

  ```console
C:\Windows\system32>torrt walk -f --verbose
DEBUG: Bootstrapping torrt environment ...
DEBUG: Importing rpc ...
DEBUG: Registering `deluge` from <class 'torrt.rpc.deluge.DelugeRPC'> ...
DEBUG: Registering `qbittorrent` from <class 'torrt.rpc.qbittorrent.QBittorrentRPC'> ...
DEBUG: Registering `transmission` from <class 'torrt.rpc.transmission.TransmissionRPC'> ...
DEBUG: Registering `utorrent` from <class 'torrt.rpc.utorrent.UTorrentRPC'> ...
DEBUG: Importing trackers ...
DEBUG: Registering `tr.anidub.com` from <class 'torrt.trackers.anidub.AniDUBTracker'> ...
DEBUG: Registering `anilibria.tv` from <class 'torrt.trackers.anilibria.AnilibriaTracker'> ...
DEBUG: Registering `casstudio.tv` from <class 'torrt.trackers.casstudio.CasstudioTracker'> ...
DEBUG: Registering `kinozal.tv` from <class 'torrt.trackers.kinozal.KinozalTracker'> ...
DEBUG: Registering `nnm-club.me` from <class 'torrt.trackers.nnmclub.NNMClubTracker'> ...
DEBUG: Registering `rutor.org` from <class 'torrt.trackers.rutor.RutorTracker'> ...
DEBUG: Registering `rutracker.org` from <class 'torrt.trackers.rutracker.RuTrackerTracker'> ...
DEBUG: Importing notifiers ...
DEBUG: Registering `email` from <class 'torrt.notifiers.mail.EmailNotifier'> ...
DEBUG: Registering `telegram` from <class 'torrt.notifiers.telegram.TelegramNotifier'> ...
DEBUG: Importing bots ...
DEBUG: Registering `telegram` from <class 'torrt.bots.telegram_bot.TelegramBot'> ...
DEBUG: Initializing objects registries from configuration file ...
DEBUG: Loading configuration file C:\Users\Toyohara\.torrt\config.json ...
DEBUG: Spawning `TransmissionRPC` object with the given settings ...
DEBUG: Registering `transmission` from transmission ...
DEBUG: Spawning `AnilibriaTracker` object with the given settings ...
DEBUG: Registering `anilibria.tv` from anilibria.tv ...
DEBUG: Spawning `RutorTracker` object with the given settings ...
DEBUG: Registering `rutor.org` from rutor.org ...
DEBUG: Spawning `RuTrackerTracker` object with the given settings ...
DEBUG: Registering `rutracker.org` from rutracker.org ...
INFO: Torrent walk is triggered
DEBUG: Loading configuration file C:\Users\Toyohara\.torrt\config.json ...
INFO: Torrent walk is started
INFO: Getting torrents from `transmission` ...
DEBUG: TransmissionRPC: RPC method `torrent-get` ...
DEBUG: Starting new HTTP connection (1): 127.0.0.1:9091
DEBUG: http://127.0.0.1:9091 "POST /transmission/rpc HTTP/1.1" 409 581
DEBUG: Starting new HTTP connection (1): 127.0.0.1:9091
DEBUG: http://127.0.0.1:9091 "POST /transmission/rpc HTTP/1.1" 200 1089
INFO:   Processing `No Mans Sky by xatab`...
DEBUG: Downloading torrent file from `https://rutracker.org/forum/viewtopic.php?t=5768151` ...
DEBUG: RuTrackerTracker: Picking a mirror ...
DEBUG: RuTrackerTracker: Probing mirror: `https://rutracker.org` ...
DEBUG: Starting new HTTPS connection (1): rutracker.org:443
DEBUG: https://rutracker.org:443 "GET / HTTP/1.1" 301 162
DEBUG: https://rutracker.org:443 "GET /forum/index.php HTTP/1.1" 200 None
DEBUG: RuTrackerTracker: Fetching https://rutracker.org/forum/viewtopic.php?t=5768151 ...
DEBUG: Starting new HTTPS connection (1): rutracker.org:443
DEBUG: https://rutracker.org:443 "GET /forum/viewtopic.php?t=5768151 HTTP/1.1" 200 None
DEBUG: RuTrackerTracker: Torrent download link found: https://rutracker.org/forum/dl.php?t=5768151
DEBUG: RuTrackerTracker: Downloading torrent file from https://rutracker.org/forum/dl.php?t=5768151 ...
DEBUG: RuTrackerTracker: Fetching https://rutracker.org/forum/dl.php?t=5768151 ...
DEBUG: Starting new HTTPS connection (1): rutracker.org:443
DEBUG: https://rutracker.org:443 "GET /forum/dl.php?t=5768151 HTTP/1.1" 200 122154
DEBUG: Torrent was downloaded from `https://rutracker.org/forum/viewtopic.php?t=5768151`
INFO:     No updates
INFO:   Processing `Frostpunk.Game.of.The.Year.Edition.GOG-InsaneRamZes`...
DEBUG: Downloading torrent file from `https://rutracker.org/forum/viewtopic.php?t=5932207` ...
DEBUG: RuTrackerTracker: Torrent download link found: https://rutracker.org/forum/dl.php?t=5768151
DEBUG: RuTrackerTracker: Downloading torrent file from https://rutracker.org/forum/dl.php?t=5768151 ...
DEBUG: RuTrackerTracker: Fetching https://rutracker.org/forum/dl.php?t=5768151 ...
DEBUG: Starting new HTTPS connection (1): rutracker.org:443
DEBUG: https://rutracker.org:443 "GET /forum/dl.php?t=5768151 HTTP/1.1" 200 122154
DEBUG: Torrent was downloaded from `https://rutracker.org/forum/viewtopic.php?t=5932207`
DEBUG:     Update is available
DEBUG: TransmissionRPC: RPC method `torrent-add` ...
DEBUG: Starting new HTTP connection (1): 127.0.0.1:9091
DEBUG: http://127.0.0.1:9091 "POST /transmission/rpc HTTP/1.1" 200 151
INFO:     Torrent is updated
DEBUG: Unregistering `1c035226e707d6712674a9048d09c888bfbe81ee` torrent ...
DEBUG: Loading configuration file C:\Users\Toyohara\.torrt\config.json ...
DEBUG: Saving configuration file C:\Users\Toyohara\.torrt\config.json ...
DEBUG: TransmissionRPC: RPC method `torrent-remove` ...
DEBUG: Starting new HTTP connection (1): 127.0.0.1:9091
DEBUG: http://127.0.0.1:9091 "POST /transmission/rpc HTTP/1.1" 200 56
INFO:   Processing `setup_rimworld_1.2.2723`...
DEBUG: Downloading torrent file from `https://rutracker.org/forum/viewtopic.php?t=5331180` ...
DEBUG: RuTrackerTracker: Torrent download link found: https://rutracker.org/forum/dl.php?t=5768151
DEBUG: RuTrackerTracker: Downloading torrent file from https://rutracker.org/forum/dl.php?t=5768151 ...
DEBUG: RuTrackerTracker: Fetching https://rutracker.org/forum/dl.php?t=5768151 ...
DEBUG: Starting new HTTPS connection (1): rutracker.org:443
DEBUG: https://rutracker.org:443 "GET /forum/dl.php?t=5768151 HTTP/1.1" 200 122154
DEBUG: Torrent was downloaded from `https://rutracker.org/forum/viewtopic.php?t=5331180`
DEBUG:     Update is available
DEBUG: TransmissionRPC: RPC method `torrent-add` ...
DEBUG: Starting new HTTP connection (1): 127.0.0.1:9091
DEBUG: http://127.0.0.1:9091 "POST /transmission/rpc HTTP/1.1" 200 151
INFO:     Torrent is updated
DEBUG: Unregistering `298f99aad2c59e7020b9779ea6808d390afa2f95` torrent ...
DEBUG: Loading configuration file C:\Users\Toyohara\.torrt\config.json ...
DEBUG: Saving configuration file C:\Users\Toyohara\.torrt\config.json ...
DEBUG: TransmissionRPC: RPC method `torrent-remove` ...
DEBUG: Starting new HTTP connection (1): 127.0.0.1:9091
DEBUG: http://127.0.0.1:9091 "POST /transmission/rpc HTTP/1.1" 200 56
DEBUG: No Notifier registered. Notification skipped
DEBUG: Loading configuration file C:\Users\Toyohara\.torrt\config.json ...
DEBUG: Saving configuration file C:\Users\Toyohara\.torrt\config.json ...
INFO: Torrent walk is finished
  ```
</details>
<details>
  <summary>config.json after update</summary>
  
  ```javascript
{
    "time_last_check": 1598469313,
    "walk_interval_hours": 24,
    "trackers": {
        "rutracker.org": {
            "username": "username",
            "query_string": null,
            "cookies": {
                "bb_session": "session",
                "bb_ssl": "1"
            },
            "password": "password"
        }
    },
    "rpc": {
        "transmission": {
            "url": "http://127.0.0.1:9091/transmission/rpc",
            "enabled": true,
            "host": "127.0.0.1",
            "user": null,
            "password": null,
            "port": "9091"
        }
    },
    "torrents": {
        "c0424991902bdba637dc1647d4b0a2869dbd1b21": {
            "hash": "c0424991902bdba637dc1647d4b0a2869dbd1b21",
            "name": "No Mans Sky by xatab",
            "url": "https://rutracker.org/forum/viewtopic.php?t=5331180",
            "url_file": "https://rutracker.org/forum/dl.php?t=5768151",
            "page": {
                "title": "No Man's Sky [P] [RUS + ENG + 11 / RUS + ENG + 10] (2016, VR Supported) (2.62 + 1 DLC) :: RuTracker.org",
                "cover": "",
                "date_updated": ""
            }
        }
    },
    "notifiers": {},
    "bots": {}
}
  ```
  
</details>
I'm using python 3.7.7 if that matters

**Comment by idlesign** [comment]
Thank you for the report.
That might be a regression issue with the lastest version. Could you try v0.13.1 to verify that?

**Comment by SakhJack** [comment]
No, they get removed from transmission

I reinstalled python at some point

The one torrent that is being preserved (No Man's Sky) is the one I added before that

I also installed torrt manually first time and using `PyPI` last time

**Comment by idlesign** [comment]
That was a regression in 0.14.0.

Seems to be fixed in 0.14.1, you can now try it out.
